eng/common/TestResources/New-TestResources.ps1

```diff
@@ -534,6 +534,7 @@ try {
 
     $tags = @{
         Creator = $UserName
```

Comment:
Is Creator still interesting?

Reply:
Probably not. I can do a sweep and remove it.